helpers/errorCodes.js

```diff
@@ -21,6 +21,26 @@ module.exports = {
 		code: 'E0004',
 		message: 'User not found'
 	},
+	E0005: {
+		code: 'E0005',
+		message: 'No courses found'
+	},
+	E0006: {
+		code: 'E0006',
+		message: 'Course not found'
+	},
+	E0007: {
+		code: 'E0007',
+		message: 'No sections not found'
```

Comment:
"No sections not found" maybe "No sections found"

Reply:
fixed